backend/src/auth/mod.rs

```diff
@@ -121,7 +126,8 @@ impl User {
         match auth_config.mode {
             AuthMode::None => Ok(Self::None),
             AuthMode::FullAuthProxy => Ok(UserData::from_auth_headers(headers, auth_config).into()),
-            AuthMode::LoginProxy => UserData::from_session(headers, db).await.map(Into::into),
+            AuthMode::LoginProxy => UserData::from_session(headers, db, auth_config.session_duration).await
+                .map(Into::into),
```

Comment:
`await` is treated like any other part in a call chain and gets its own line. At least thats how the rest of the code does it everywhere.

Reply:
*grumble mumble*